src/renderer/components/NewTeamModal.tsx

```diff
@@ -27,11 +29,13 @@ type State = {
     teamIndex?: number;
     teamOrder: number;
     saveStarted: boolean;
+    errorMessage: string;
```

Comment:
is this being used somewhere? or are we only storing the value?

Reply:
In the previous implementation, I considered perhaps displaying some alert about the server not existing. Since now I only ping when the form is saved, this property is pointless, so I removed it. I simply log the message for debugging purposes.